src/Compilers/CSharp/Test/Syntax/Parsing/DeclarationParsingTests.cs

```diff
@@ -3014,7 +3014,7 @@ public void TestClassMethodWithMultipleParameters()
             Assert.Equal("b", ms.ReturnType.ToString());
             Assert.NotEqual(default, ms.Identifier);
             Assert.Equal("X", ms.Identifier.ToString());
-            Assert.NotNull(ms.ParameterList.OpenParenToken);
+            Assert.NotEqual(default, default, ms.ParameterList.OpenParenToken);
```

Comment:
😕 Why two copies of `default,`?

Reply:
Cause I double tapped the key bound to my vim macro to fix this thing. This is in an old commit though, not in the HEAD of the PR.